core/build.sbt

```diff
@@ -5,3 +5,7 @@ name := "slinky-core"
 libraryDependencies += "org.scala-lang" % "scala-reflect" % scalaVersion.value
 
 scalacOptions += "-P:scalajs:sjsDefinedByDefault"
+
+resolvers += Resolver.sonatypeRepo("releases")
+
+addCompilerPlugin("org.typelevel" % "kind-projector" % "0.10.3" cross CrossVersion.binary)
```

Comment:
What is this needed for? I really want to avoid using fancy type tricks if it's not absolutely necessary.

Reply:
This is to support Either, Map, Tuple types which take two arguments instead of one.

Happy to not support these types. It just means you can't pass say a Map[String, ReactElement] between components. But that is probably not that common anyway.